### ISSUE TITLE
Make the grid on the rag report dashboard ajax loaded

### ISSUE THREAD

**Issue by fritogotlayed** [MadDonkeySoftware/RAG-Report#38]
Depends upon #37 being merged.

On http://127.0.0.1:8000/rag make the grid loaded via an ajax call when the "Apply" button is pressed. This should include updating the URL in the browser so that users can copy-paste the link if needed. The URL changing can be done through https://github.com/browserstate/history.js/

**Comment by MrsDbones** [comment]
Nice to have, not needed, remove from project
